src/test/java/hw1/CalculatorDataProvider.java

```diff
@@ -0,0 +1,52 @@
+package hw1;
+
+import org.testng.annotations.DataProvider;
+
+public class CalculatorDataProvider {
```

Comment:
Why do you decide choose these set of test data?

Reply:
I decided to choose this set of data because it's very easy to select and change different test data. At first I tried to use *double*-typed operations, but I had some problems with their accuracy (e.g. some operations give the result like 5.999999997 instead of 6), so I chose the *long*-typed operations